_scalingo_api/events.md

```diff
@@ -1771,6 +1771,99 @@ Example object:
   }
 }
 ```
+--- row ---
+
+* **Database Maintenance Planned**
+
+_When:_ A maintenance is scheduled on your database
+`type=plan_database_maintenance`
+
+{:.table}
+| field                         | type      | description                     |
+| ------------------------------| --------- | ------------------------------- |
+| maintenance_id                | string    | The related maintenance ID      |
+| addon_name                    | string    | The related addon name          |
+
+||| col |||
+
+Example object:
+
+```json
+{
+  "id":"62d13ad1a5b6ec0001e553c8",
+  "created_at":"2022-07-15T10:00:49.703Z",
+  "type":"plan_database_maintenance",
+  "app_id": "5343eccd646173000a140000",
+  "app_name": "appname",
+  "type_data": {
+    "maintenance_id": "65004006a8acb50076cf9c1b",
+    "addon_name": "redis"
```

Comment:
As explained above, we want to add all available information,
i.e.
```json
            "type_data": {
                "addon_name": "Redis",
                "maintenance_id": "65004006a8acb50076cf9c1b",
                "maintenance_type": "db-node-migration",
                "maintenance_duration": 8,
                "next_maintenance_window": "2023-09-14T21:00:00.000Z"
            }
```
And `user`:
```json
  "user": {
    "username": "johndoe",
    "email": "john@doe.com",
    "id": "us-0e6d8e46-5cd0-42a4-acba-372b2be605ac"
},
```